### ISSUE TITLE
"Loading Sections" blocks usage

### ISSUE THREAD

**Issue by MrMallIronmaker** [cs-education/ClassCapture_AndroidApp#4]
If the backend server is down when opening the app, "Loading Sections" pops up and blocks the user for ten seconds. As with #3, I'm not certain it's the backend server being down, but that will be verified shortly.

**Comment by sourabhdesai** [comment]
Yes, it will block while the request for sections is being made. In the normal case, the request wont take very long and the modal dialog will go away quickly.

Requests will be blocked if:
* User isn't on a UIUC Wifi Network or logged in via VPN
* Backend call lags when proxying API request from WebApp server to API server.

The 2nd bullet point is currently an issue that needs to be solved. The first point will remain an issue until we are ready to release the app from the privately hosted UIUC network.

For the time being, make sure that you are on a UIUC wifi or are using VPN. You shouldn't be seeing this issue with the latest APK file I emailed you.

Android VPN instructions here: https://answers.uillinois.edu/illinois/page.php?id=47644
